test/src/org/labkey/test/tests/targetedms/TargetedMSChromatogramOptimizationTest.java

```diff
@@ -0,0 +1,133 @@
+package org.labkey.test.tests.targetedms;
+
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.WebTestHelper;
+import org.labkey.test.categories.DailyB;
+import org.labkey.test.categories.MS2;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.targetedms.ConnectionSource;
+
+import java.io.File;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Category({DailyB.class, MS2.class})
```

Comment:
Panorama/TargetedMS is separate from the MS2 module. I know other TargetedMS tests are tagged as MS2. I don't know what the suite is intended to represent, but it should be renamed or slimmed down.

Reply:
Removed..!